content/4-component-composition/5-context/solid/UseContext.jsx

```diff
@@ -0,0 +1,3 @@
+import { createContext } from "solid-js";
```

Comment:
I think it's better to call the file `UserContext.jsx` instead of `UseContext.jsx`

Reply:
I originally named it Use rcontext.jsx, but I accidentally typed one less "r"😅, and I'm going to change it.